ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -603,6 +614,35 @@ FAN_SS_VSC_FDPM_DRSM2X:
   - FDPM
   - DRSM2X
 
+FAN_SS_VSC_COM:
+  guid: "4747b38a-c746-4056-bc04-3920d8734fa5"
+  description: "Fan with start/stop status, variable speed control and CO monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - COM
+
+FAN_SS_VSC_IDC:
+  guid: "459e5ea9-859e-4bf8-95a5-95c24e4f6af7"
+  description: "Fan with start/stop status, variable speed control and isolation damper control."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - IDC
+
+FAN_SS_VSC_REFM: # EF 0-09, SF 0-01
+  description: "Fan with start/stop status, variable speed control and refrigerant monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - REFM
```

Comment:
not refrigerant concentration control?

Reply:
Confirmed in the BMS that refrigerant sensor is just used for alarms.